src/content/mod_configuration.cpp

```diff
@@ -230,18 +230,34 @@ void ModConfiguration::resolveDependencies()
 	// of each mod, split mods into satisfied and unsatisfied
 	std::vector<ModSpec> satisfied;
 	std::list<ModSpec> unsatisfied;
+	ModSpec firstModSpec;
 	for (ModSpec mod : m_unsatisfied_mods) {
+		// dependencies are not allowed for first ad last mod
+		if ( (mod.name == first_mod) || (mod.name == last_mod) ) {
+			if (!mod.depends.empty() || !mod.optdepends.empty())
+				throw ModError("Mod selected as first/last by game is not allowed to have dependencies.");
```

Comment:
Why have this restriction for the last mod? Since it loads last, it naturally can depend on anything (though that needn't be useful).

Reply:
Fixed.